### PR TITLE
macOS build 

### PR DESCRIPTION
**Pull request by Foxushka**

### CODE REVIEW

**Review by GameTec-live — CHANGES_REQUESTED** [review]

File: .github/workflows/build-app.yml
[the remark this answers is not in the document]
Comment (Foxushka) [reply .github/workflows/build-app.yml:1]: No, won't. Release workflow is even in other file, so can't cause any issues

[the remark this answers is not in the document]
Comment (GameTec-live) [reply .github/workflows/build-app.yml:1]: theres no benefit to renaming the file and resetting the "run" counter just makes a major build number gap

[the remark this answers is not in the document]
Comment (Foxushka) [reply .github/workflows/build-app.yml:1]: Benefit - consistent file naming

[the remark this answers is not in the document]
Comment (GameTec-live) [reply .github/workflows/build-app.yml:1]: noone cares about that 🤷

[the remark this answers is not in the document]
Comment (Foxushka) [reply .github/workflows/build-app.yml:1]: Why someone should care about run counter then?

**Review by GameTec-live — APPROVED** [review]

**Pull request merged**